content/notes/new-site.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "From openrevolution.net to diginomics.io - why we are updating the site"
+created: 2023-04-27 
+authors: [ theo-cox, rufuspollock]
+type: Blog
+---
+
+
+You might have noticed that the open revolution website has evolved. We realized that we had a lot of material (and even more thinking) which we hadn’t published anywhere, and that there was a clear need for more generalized and practically focused content. We created the new diginomics site to address this need. The site provides grounding in the economics of digital (information) economy and toolkits of concepts and templates for solving key micro and macro challenges e.g. sustaining data sharing in X area (e.g. carbon data) or addressing tech monopolies.
+
+This blog lays out our thinking behind the evolution in the SCQA format.
+
+
+# Situation
+The openrevolution.net site and book have existed for some time. The content has been well received but the book tells a high level story. The existing openrevolution.net site focuses on the book; it’s currently just “the book website”. 
+
+The topics covered by the book continue to become ever more visible and mainstream (Google, Apple, Facebook, Amazon etc are now clearly semi “bad guys”). We are getting approached by people involved in various projects related to data sharing who want insight into the economics and incentives to underpin this; understanding is limited and there is a need.
+
+We have other material both published (e.g. imedproject) or unpublished that is relevant, which could be helpful if it was more accessible to people.
+
+
+# Complication
```

Comment:
`## The Need and Opportunity: there's a broader topic to address and we already have the material`